.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: python
+python:
+  - "2.7"
+env:
+  - DJANGO=1.4.20
```

Comment:
I originally included 1.2 and 1.3, but they failed pretty spectacularly.
1.7 and 1.8 are also failing but fixing 1.7 compatibility is outside of the scope of *this* PR ('though it *does* let us see that 1.7 and 1.8 have 'issues')